smartsim/exp/ray/raycluster.py

```diff
@@ -442,22 +522,35 @@ def _build_ray_exe_args(self, ray_password, ray_args, ray_port, interface):
 
         return " ".join(ray_starter_args)
 
-    def _build_run_settings(self, launcher, alloc, run_args, ray_exe_args):
+    def _build_run_settings(self, launcher, alloc, run_args, ray_exe_args, run_command):
```

Comment:
I think we could safely migrate this to use the ``settings.create_run_settings`` method. and the also the ``create_batch_settings`` method when that goes in.